rllib/core/learner/learner.py

```diff
@@ -110,10 +108,7 @@ class LearnerHyperparameters:
     respective AlgorithmConfig class.
     """
 
-    # TODO (Sven): Move lr from - currently - optimizer config to only exist here.
-    # lr: float = None
-
-    lr_schedule: Optional[List[List[Union[int, float]]]] = None
+    pass
```

Comment:
see above: moved lr_schedule back into optimizer_config dict. Need tho think about this more and come up with a final comprehensive solution for optimizer configs vs learner hps.